src/components/HydrantList.js

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import List, { ListItem, ListItemText } from 'material-ui/List';
+import Delete from '@material-ui/icons/Delete';
+import ExpandLess from '@material-ui/icons/ExpandLess';
+import ExpandMore from '@material-ui/icons/ExpandMore';
+import Collapse from 'material-ui/transitions/Collapse';
+import Input, { InputLabel } from 'material-ui/Input';
+import { withStyles } from 'material-ui/styles';
+import ModeEdit from '@material-ui/icons/ModeEdit';
+import { FormControl, FormHelperText } from 'material-ui/Form';
+import TextField from 'material-ui/TextField';
+import _ from 'lodash';
+
+
+const styles = theme => ({
+  root: {
+    flexWrap: 'wrap'
+  },
+  inset: {
+    paddingLeft: 0
+  }
+});
+
+
+
+class HydrantList extends React.Component {
+
+  state = {
+    open: true,
+    showDetails:{}
+  }
+
+render() {
```

Comment:
Linter: things inside the class should be indented.

Reply:
Check